conf/formTemplateSchema.json

```
@@ -469,6 +469,13 @@
             }
           ]
         },
+        "descriptionTotal": {
+          "anyOf": [
+            {
+              "$ref": "#/$defs/atlDescriptionObject"
+            }
+          ]
```

```suggestion
          "$ref": "#/$defs/atlDescriptionObject"
```

Can't this be just single `$ref`